build/ci/golangci.yml

```diff
@@ -63,7 +63,7 @@ linters:
     - errorlint
     - exportloopref
     - forbidigo
-    #- gci
+    - gci
```

Comment:
Cache jsem vymazal, tak uvidíme. Je pravda že lokálně mi to taky prochází.

Reply:
Ok. Veď ak by s tým boli problémy tak vyriešime neskôr.